src/test/ui/issues/issue-53675-a-test-called-panic.rs

```diff
@@ -0,0 +1,35 @@
+// rust-lang/rust#53675: At one point the compiler errored when a test
+// named `panic` used the `assert!` macro in expression position.
+
+// compile-flags: --test
```

Comment:
Missing `compile-pass`.

Reply:
Face palm. Will fix.

Reply:
(Well, I did make it work as a run-pass test I think. But I don’t mind it being compile-pass.)